.github/pull_request_template.md

```diff
@@ -0,0 +1,88 @@
+<!-- 
+Thank you for contributing to terraform-provider-aap!
+
+Please review our contribution guidelines: 
+https://github.com/ansible/terraform-provider-aap/blob/main/CONTRIBUTING.md
+-->
+
+## Description
+
+<!-- What does this PR do? What issue does it fix? -->
+
+**Related Issue:** <!-- Link to GitHub issue if applicable -->
+
+## Type of Change
+
+- [ ] Bug fix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Documentation update
+- [ ] Code refactoring (no functional changes)
+
+## Testing Checklist
+
+### Unit Tests
+- [ ] Unit tests added/updated for new code
+- [ ] All unit tests pass locally (`make test`)
+- [ ] Code coverage ≥80% for new code (`make testcov`)
+
+### Acceptance Tests
+- [ ] Acceptance tests added/updated
+- [ ] Acceptance tests pass (see output below)
```

Comment:
Did we decide contributors needed to run acceptance tests or that we would as reviewers?